src/Text/Markdown/SlamDown/Html.purs

```diff
@@ -215,6 +217,13 @@ evalSlamDownQuery e =
     GetFormState k -> do
       SlamDownState state <- H.get
       pure $ k state.formState
+    PopulateForm values next -> do
+      SlamDownState { document } <- H.get
+      let desc = formDescFromDocument document
+          keysToPrune = filter (`M.member` desc) (toList (M.keys values))
+          prunedFormState = foldr M.delete values keysToPrune
+      H.modify <<< modifySlamDownState $ const prunedFormState
```

Comment:
Is it possible to replace the `get` followed by `modify` with a single call to `modify` without `const`?